packages/frontend-shared/src/utils/spec-utils.ts

```diff
@@ -118,35 +120,13 @@ export function makeFuzzyFoundSpec (spec: FoundSpec): FuzzyFoundSpec {
   }
 }
 
-export type SpecsComparator<T extends Readonly<FileLike>> = (
-  specs: Readonly<T[]>,
-  oldSpecs: Readonly<T[]>
-) => boolean
-
-type FileLike = {
-  absolute: string
-}
-
-function defaultCompareFn (
-  specs: Readonly<Array<FileLike>>,
-  oldSpecs: Readonly<Array<FileLike>>,
-) {
-  return specs.some((spec, idx) => spec.absolute !== oldSpecs[idx]?.absolute)
-}
-
 export function useCachedSpecs<S extends { absolute: string }> (
   specs: ComputedRef<Readonly<S[]>>,
-  compareFn?: SpecsComparator<S>,
 ): Ref<Readonly<S[]>> {
   const cachedSpecs: Ref<Readonly<S[]>> = ref([])
 
   watch(specs, (currentSpecs, prevSpecs = []) => {
```

Comment:
It feels like `cachedSpecs` should become a computed property...
I might be mistaken.